test/unit/testDocumentsPage.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const initSteps = require('app/core/initSteps');
+const chai = require('chai');
+const expect = chai.expect;
+const steps = initSteps([`${__dirname}/../../app/steps/action/`, `${__dirname}/../../app/steps/ui`]);
+
+describe('ApplicantAliasReason', () => {
+    const Documents = steps.Documents;
+
+    describe('getUrl()', () => {
+        it('should return the correct url', (done) => {
+            const url = Documents.constructor.getUrl();
+            expect(url).to.equal('/documents');
+            done();
+        });
+    });
+});
```

Comment:
Can we also have a test for `handleGet()` for at least the addition of `ctx.executorsNameChangedByDeedPollList`

Reply:
Unit testing this function raises an issue we have encountered in the past as a result of this call: const content = this.generateContent(ctx, formdata);
It causes the test in the payments file to fail. The executorsWrapper.executorsNameChangedByDeedPoll() has enough test coverage to make up for this.